src/java/picard/analysis/CollectWgsMetrics.java

```diff
@@ -158,6 +165,7 @@ protected int doWork() {
         final int max = COVERAGE_CAP;
         final long[] HistogramArray = new long[max + 1];
         final long[] baseQHistogramArray = new long[Byte.MAX_VALUE];
+        final long[] baseQHetSensHistogram = new long[Byte.MAX_VALUE];
```

Comment:
Put in a comment as to why a separate base quality histogram is needed for het sensitivity.